src/table.rs

```diff
@@ -237,6 +274,11 @@ impl ValueTable {
 				return Ok(false);
 			}
 
+			if key.is_none() && &buf[0..2] == INIT {
```

Comment:
0-sized tables and 0-size values are legitimate. `INIT` might match a 0-sized value.

Reply:
I did notice that (see comment), this needs to change the format for multipart to work.

Reply:
This is not a problem with multipart, but with fixed-size tables as well. 0-sized entries are allowed in fixed size tables.

Reply:
Oh, I remember: the size for those entries does include the size the other fields, so I was thinking it could not be 0.